src/commands/avatar.rs

```diff
@@ -0,0 +1,15 @@
+use crate::Context;
+use color_eyre::eyre::Result;
+use poise::serenity_prelude as serenity;
+
+/// Displays your or another user's avatar
+#[poise::command(slash_command)]
+pub async fn avatar(
+    ctx: Context<'_>,
+    #[description = "Selected user"] user: Option<serenity::User>,
+) -> Result<()> {
+    let u = user.as_ref().unwrap_or_else(|| ctx.author());
+    let av = u.avatar_url().expect("Could not get avatar URL");
```

Comment:
Why are you obfuscating variable names this is not C

Reply:
go habbits tbh